why-oc.md

```diff
@@ -57,6 +71,107 @@ So now lets take a look at OpenCore's method:
 
 Things to note with OpenCore's method is:
 
-* OS Agnostic as the prelinked kernel format as stayed the same since  10.6, far harder to break support.
-  * OpenCore also supports cacheless, Mkext and KernelCollections meaning it also has proper support for all Intel versions of OS X/macOS. However proper support has been added at 10.6 and newer.
+* OS Agnostic as the prelinked kernel format as stayed the same since 10.6(V2), far harder to break support.
+  * OpenCore also supports prelinked kernel(V1, found in 10.4 and 10.5), cacheless, Mkext and KernelCollections meaning it also has proper support for all Intel versions of OS X/macOS
 * Far better stability as far less patching
+
+# Common Myths
+
+## Is OpenCore unstable as it's a beta
+
+Short Answer: No
+
+Long Answer: No
+
+OpenCore's version number does not represent the quality of the project. Instead, it's more of a way to see the stepping stones of the project. Acidanthera still has much they'd like to do with the project including overall refinement and more feature support.
+
+For example, OpenCore goes through proper security audits to ensure it complies with UEFI Secure Boot, and is the only Hackintosh bootloader to undergo these rigorous reviews and have such support.
+
+Version 0.6.1 was originally designed to be the official release of OpenCore as it would have proper UEFI/Apple Secure Boot, and would be the 1 year anniversary of OpenCore's release as a public tool. However, due to circumstances around macOS Big Sur and the rewriting of OpenCore's prelinker to support it, it was decided to push of 1.0.0 for another year.
```

Comment:
```suggestion
Version 0.6.1 was originally designed to be the official release of OpenCore as it would have proper UEFI/Apple Secure Boot, and would be the 1 year anniversary of OpenCore's release as a public tool. However, due to circumstances around macOS Big Sur and the rewriting of OpenCore's prelinker to support it, it was decided to push off 1.0.0 for another year.
```